src/Categories/Category/Construction/F-Coalgebras.agda

```diff
@@ -0,0 +1,129 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.F-Coalgebras where
+
+open import Level
+
+open import Categories.Category
+open import Categories.Functor hiding (id)
+open import Categories.Functor.Algebra
+open import Categories.Functor.Coalgebra
+open import Categories.Object.Initial
+open import Categories.Object.Terminal
+import Categories.Morphism.Reasoning as MR
+import Categories.Morphism as Mor using (_≅_; Iso)
+open import Categories.Category.Construction.F-Algebras
+open import Categories.Functor.Duality
+
+private
+  variable
+    o ℓ e : Level
+    C : Category o ℓ e
+
+F-Coalgebras : {C : Category o ℓ e} → Endofunctor C → Category (ℓ ⊔ o) (e ⊔ ℓ) e
+F-Coalgebras {C = C} F = record
+  { Obj       = F-Coalgebra F
+  ; _⇒_       = F-Coalgebra-Morphism
+  ; _≈_       = λ α₁ α₂ → f α₁ ≈ f α₂
+  ; _∘_       = λ α₁ α₂ → record
+    { f = f α₁ ∘ f α₂
+    ; commutes = F-Algebra-Morphism.commutes (F-Coalgebra-Morphism⇒coF-Algebra-Morphism α₁
+       coF-Algebras.∘
+       F-Coalgebra-Morphism⇒coF-Algebra-Morphism α₂)
+    }
+  ; id        = coF-Algebra-Morphism⇒F-Coalgebra-Morphism coF-Algebras.id
```

Comment:
I fear that this remains a bad idea, i.e. you are better off defining `id` directly. `commutes` above is ok.

Reply:
Done c789ca7d